observatory-platform/observatory/platform/elastic/kibana.py

```diff
@@ -75,23 +56,36 @@ class ObjectType(Enum):
 class Kibana:
     HTTP_NOT_FOUND = 404
 
-    headers = {"Content-Type": "application/json", "kbn-xsrf": "true"}
+    headers: dict = None
```

Comment:
This should be a parameter in the constructor

Reply:
Thanks, just changed this